data/field_config_test.go

```diff
@@ -0,0 +1,67 @@
+package data_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReadMappings(t *testing.T) {
+	jsonText := `{
+		"description": "turn on/off system. write 1 to turn on the system and write 0 to turn off the system",
+		"writeable": true,
+		"mappings": [
+		  {
+			"text": "OFF",
+			"value": "0"
+		  },
+		  {
+			"type": 1,
+			"text": "ON",
+			"value": "1"
+		  },
+		  {
+			"type": 2,
+			"text": "0-100",
+			"from": "0",
+			"to": "100"
+		  }
+		]
+	}`
+
+	cfg := &data.FieldConfig{}
+	err := json.Unmarshal([]byte(jsonText), &cfg)
+	assert.NoError(t, err, "error parsing json")
```

Comment:
It's a better idea to halt the test on error via `require`, since continuing (via `assert`) will be chaotic.
```suggestion
	require.NoError(t, err, "error parsing json")
```